lib/cabalProjectToNix.nix

```diff
@@ -29,6 +29,8 @@ let
     # without the source available (we cleaneSourceWith'd it),
     # this may not produce the right result.
     find . -name package.yaml -exec hpack "{}" \;
+    export SSL_CERT_FILE=${cacert}/etc/ssl/certs/ca-bundle.crt
```

Comment:
Shouldn't we put these into the `nix-tools` expression as well?

Reply:
Maybe, I don't know which command needs `SSL_CERT_FILE`.

I thought I wrote a comment here before saying that I started the fix with the above commit, but it's probably not fully complete.

Reply:
IIRC it was nix-prefetch-git that needed SSL_CERT_FILE and I think it still worked, but printed a warning that it did not validate the cert.  We need to make sure we validate the cert since the hash we compute here is the one we trust to be correct.

Reply:
Yes, i would expect nix-prefetch-git to be the one that calls out to curl.

Reply:
I think `SSL_CERT_FILE` and `GIT_SSL_CAINFO` should _not_ be in the nix-tools wrapper scripts. The reason they are required is probably because `/etc/ssl/certs` can't be accessed from the nix build sandbox.

Reply:
Fair point. But does `nix-shell --pure -p nix-tools` work with source repo cabal projects then?

Reply:
Yes, the pure shell works. I just tested `plan-to-nix` with the `cabal.project` from nix-tools.